src/components/lab/GridScale.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Rect, Group, Text } from 'react-konva';
+import PropTypes from 'prop-types';
+import {
+  BACKGROUND_COLOR,
+  GRID_AXES_COLOR,
+  GRID_AXES_STROKE_WIDTH,
+  GRID_LEGEND_LABEL_TEXT,
+  GRID_SCALE_FONT_SIZE,
+  GRID_SCALE_TEXT_WIDTH,
+  GRID_SQUARE_WIDTH_AND_HEIGHT,
+} from '../../config/constants';
+
+const SCALE_STROKE_MARGIN = 2;
```

Comment:
Minor but maybe nicer to have this with the rest of the constants in `constants.js`, along with the numerical props in `<Text>` below